Server/src/services/helpers/validationSchema.ts

```diff
@@ -4,6 +4,9 @@ const integerMessage = (fieldName: string): string => {
     return `${fieldName} should be an integer`
 }
 
+const integerOrNullMessage = (fieldName: string): string => {
+    return `${fieldName} should be an integer or null`
```

Comment:
Im not sure if this is valid. You are passing in a string but checking if its an integer. Either make the fieldname an integer or change the function name

Reply:
A regex check is required as there is no native way in Yup to check for integer or null and Yup won't accept regex checks versus an integer and that is why it is casted to a string for this specific check. 
However, on the front end side, it will always check for an integer, so the only case where a string could be inputted would be with a direct backend upload, but those will fail the backend validation as it will fail the regex check. I would contend that it is valid on these grounds.

Reply:
try .integer().nullalble() will do the job

Reply:
That does not work; already tried it.